src/components/material/Card.vue

```diff
@@ -1,116 +0,0 @@
-<template>
```

Comment:
Oh noes! Found that this one is actually used. Could you revert this change, please @garrlker ? My bad.

Here's where it is used: https://github.com/cylc/cylc-ui/blob/master/src/views/GScan.vue#L14

Found after seeing a browser console error.

Reply:
Screenshot: 

![image](https://user-images.githubusercontent.com/304786/66011052-fd6b7100-e51d-11e9-83f4-854358eb5bb8.png)

Reply:
Oh dang!

I just put it back and added myself to the `contributing.md`.
I also went ahead and resolved a merge conflict with `contributing.md`.

Everything should be good to go! 👍